taskcluster/translations_taskgraph/transforms/training_continuation.py

```diff
@@ -0,0 +1,94 @@
+from taskgraph.transforms.base import TransformSequence
+from urllib.parse import urljoin
+import os
+
+CONTINUE_TRAINING_ARTIFACTS = (
+    "devset.out",
+    "model.npz",
+    "model.npz.best-bleu-detok.npz",
+    "model.npz.best-bleu-detok.npz.decoder.yml",
+    "model.npz.best-ce-mean-words.npz",
+    "model.npz.best-ce-mean-words.npz.decoder.yml",
+    "model.npz.best-chrf.npz",
+    "model.npz.best-chrf.npz.decoder.yml",
+    "model.npz.decoder.yml",
+    "model.npz.optimizer.npz",
+    "model.npz.progress.yml",
+    "model.npz.yml",
+    "train.log",
+    "valid.log",
+    "vocab.spm",
+)
+
+INITIALIZE_MODEL_ARTIFACTS = (
+    "model.npz.best-bleu-detok.npz",
```

Comment:
There should be final.* name. It's the file we copy in the end which might not be the case for older models. Also only one model file is required for initialization.